integreat_cms/core/signals/hix_signals.py

```diff
@@ -35,16 +35,15 @@ def page_translation_save_handler(instance, **kwargs):
         instance.hix_score = None
         return
 
-    if identical_version := instance.all_versions.filter(
-        content=instance.content, hix_score__isnull=False
-    ).first():
+    latest_version = instance.latest_version
+
+    if latest_version and latest_version.content == instance.content:
```

Comment:
Otherwise we won't be able to obtain the stable HIX value without changing the page content.
Because if previous equal content with an old (broken) value is found, it will be constantly re-stored instead of sending a new request.
Alternatively, we can make a management command to fix the "broken" values?

Reply:
I think this way is fine 🙂